bin/stacks/routing-caching-stack.ts

```diff
@@ -37,6 +47,13 @@ export class RoutingCachingStack extends cdk.NestedStack {
       ],
     });
 
+    lambdaRole.addToPolicy(
+      new PolicyStatement({
+        resources: [route53Arn!],
```

Comment:
Does it still deploy if route53Arn is undefined? I think IAM does some basic validation of policies during deployment so I would expect the deployment to fail?

Reply:
Ok will do!

Reply:
Yeah, we'd need to require the route53 arn so I just wrapped it in an if statement